test/cases/custom-loader-with-functional-exports/expected/main.js

```diff
@@ -8,12 +8,14 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony export */ __webpack_require__.d(__webpack_exports__, {
 /* harmony export */   cnA: () => (/* binding */ _1),
-/* harmony export */   cnB: () => (/* binding */ _2)
+/* harmony export */   cnB: () => (/* binding */ _2),
+/* harmony export */   "default": () => (__WEBPACK_DEFAULT_EXPORT__)
```

Comment:
We still need an option for this, otherwise we generate extra content, it is bad for a size

Reply:
/cc @jantimon